drivers/timer/gecko_burtc_timer.c

```diff
@@ -69,6 +69,8 @@ static atomic_t g_last_count;
 /* Spinlock to sync between Compare ISR and update of Compare register */
 static struct k_spinlock g_lock;
 
+/* Set to true when timer is initialized */
+static atomic_t g_init = ATOMIC_INIT(0);
```

Comment:
Out of curiosity, why are you using `atomic_t` instead of just `bool`? I don't see you doing any "compare and set" style operations, which would justify the overhead of more protected access to the value.

Reply:
You're right, it was something remaining from initial prototyping. Changed to `bool`